solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -245,6 +245,8 @@ contract RandomBeacon is Ownable {
 
     event CallbackFailed(uint256 entry, uint256 entrySubmittedBlock);
 
+    event RewardsBanned(uint32[] ids, uint256 duration);
+
     event BanRewardsFailed(uint32[] ids);
```

Comment:
Let's drop these two events. We should assume sortition pool never reverts on marking operators ineligible for rewards and even if it does, it is fine to revert DKG submit transaction because not submitting the result does not lead to slashing.

Reply:
See https://github.com/keep-network/keep-core/pull/2729/commits/ef1075cc3260578686539f43fba5bd11fc6b5a0a